application/controllers/Users.php

```diff
@@ -0,0 +1,111 @@
+<?php
+class Users extends CI_Controller {
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        // This line will load user model to this controller
+        $this->load->model('users_model');
+
+        $this->load->helper('url');
+    }
+
+    public function index()
+    {
+        $this->load->helper('form');
+		// Use method get_users of user model to get users list
+        $data["users"] = $this->users_model->get_users();
+
+        // declaring page_title variable
+        $data["page_title"] = "List Of Users";
+
+
+        $this->load->view('header', $data); 
+
+        // assign users list to view
+        $this->load->view('users/index', $data);
+        $this->load->view('footer');
+    }
+
+    public function create()
+    {
+        $this->load->helper('form');
+        $this->load->library('form_validation');
+        $this->load->helper('url');
+
+        $data["page_title"] = "Create New User";
+        $this->form_validation->set_rules('first_name', 'First name', 'required');
+        $this->form_validation->set_rules('last_name', 'Last name', 'required');
+        $this->form_validation->set_rules('email', 'Email', array('required','valid_email'));
+		$this->form_validation->set_rules('phone_number', 'Phone number', 'required');
```

Comment:
There are some missing indentations in your source code, please check all files ad fix it.
You should enable Rendering White Spaces in VSCOde to avoid confusion between white spaces and tabs
<img width="288" alt="Screen Shot 2021-06-11 at 08 33 20" src="https://user-images.githubusercontent.com/28641693/121619313-97751200-ca92-11eb-90e7-57ff0b9b0e61.png">

Reply:
I fixed an unorganized indentation & arranged it into 4 spaces.